openwisp_monitoring/device/migrations/0009_update_device_status_for_disabled_critical_checks.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+from swapper import load_model
+
+
+def update_critical_device_metric_status(apps, schema_editor):
+    Check = apps.get_model('check', 'Check')
+    DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
```

Comment:
Add a comment above here explaining why we need the concrete model here. Eg:

```suggestion
    # We need to load the real concrete model here, because we 
    # will be calling one of its class methods below
    DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
```